templates/committee_manager/list_members.html.twig

```diff
@@ -169,6 +169,9 @@
                             {% else %}
                                 Membre
                             {% endif %}
+                            {% if member.id in committee.activeAdherentMandateAdherentIds %}
```

Comment:
could you move `committee.activeAdherentMandateAdherentIds` call outside the loop?